includes/handler/class-move.php

```diff
@@ -41,64 +42,64 @@ public static function handle_move( $activity ) {
 			return;
 		}
 
-		$target_object = Http::get_remote_object( $target );
-		$origin_object = Http::get_remote_object( $origin );
+		$target_actor_object = Http::get_remote_object( $target );
+		$origin_actor_object = Http::get_remote_object( $origin );
 
-		$verified = self::verify_move( $target_object, $origin_object );
+		$verified = self::verify_move( $target_actor_object, $origin_actor_object );
 
 		if ( ! $verified ) {
 			return;
 		}
 
-		$target_follower = Followers::get_follower_by_actor( $target );
-		$origin_follower = Followers::get_follower_by_actor( $origin );
+		$target_actor_cpt = Actors::get_remote_by_uri( $target );
+		$origin_actor_cpt = Actors::get_remote_by_uri( $origin );
```

Comment:
```suggestion
		$target_actor = Actors::get_remote_by_uri( $target );
		$origin_actor = Actors::get_remote_by_uri( $origin );
```
Removing the suffix might help with making it visually easier to read without losing meaning

Reply:
I thought to be consistent with `target_actor_object`!?!